src/mlpack/methods/linear_svm/linear_svm_main.cpp

```diff
@@ -0,0 +1,417 @@
+/**
+ * @file linear_svm_main.cpp
+ * @author Yashwant Singh Parihar
+ *
+ * Main executable for linear svm.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/util/cli.hpp>
+#include <mlpack/core.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+
+#include "linear_svm.hpp"
+
+#include <ensmallen.hpp>
+
+using namespace std;
+using namespace mlpack;
+using namespace mlpack::svm;
+using namespace mlpack::util;
+
+PROGRAM_INFO("LinearSVM is an L2-regularized support vector machine model",
+    // Short description.
+    "An implementation of LinearSVM for multiclass classification. "
+    "Given labeled data, a model can be trained and saved for "
+    "future use; or, a pre-trained model can be used to classify new points.",
+    // Long description.
+    "An implementation of LinearSVM using either the "
```

Comment:
I would suggest changing `An implementation of LinearSVM using` to `An implementation of linear SVMs that uses either L-BFGS or parallel SGD (stochastic gradient descent) to train the model.`, since the optimizers are only used for training. :+1:

Also, note that the typical user won't be familiar with the internal C++ classes like `LinearSVM` or `ParallelSGD`, so maybe it is better to use their "logical names", i.e. `linear SVM` and `parallel SGD`.

Reply:
Updated.